schemas/overview.md

```diff
@@ -43,8 +43,9 @@ All of this maps over the IPLD Data Model:
 
 - every type has a representation;
 - e.g. maps are represented as maps; structs are also represented as maps!
-- representations can specify aliases for how to serialize things; e.g. provide
-  shorter serial strings to use in place of longer struct names.
+- representations can specify aliases (referred to as a "rename" in the schema
```

Comment:
Oh, boy, that's stilted now.

Maybe we can try out some phrasing using the word "remapped"?  Idea spawned by looking at other text elsewhere, also commented there.

Reply:
> - representations can specify a mapping for serialized field names, which can
>  be useful in cases such as requiring serialization compactness but preferring
>  verbosity in the schema for descriptive purposes.